easybuild/easyblocks/b/boost.py

```diff
@@ -89,6 +89,8 @@ def patch_step(self):
     def configure_step(self):
         """Configure Boost build using custom tools"""
 
+        configopts = "--prefix=%s" % self.objdir
```

Comment:
use `self.cfg['configopts'].update("--prefix=%s" % self.objdir)`, and drop `configopts` (same below)